help/destinations/catalog/advertising/magnite-streaming.md

```diff
@@ -0,0 +1,153 @@
+# Magnite Streaming: Real-Time Destination
+
+[Overview](#overview)
+
+[The Magnite-Streaming: Real-Time Destination](#the-magnite-streaming-real-time-destination)
+
+[Prerequisites](#prerequisites)
+
+[Supported Identities](#supported-identities)
+
+[Setting Up The
+Destinations](#setting-up-the-destinations)
+
+&ensp;&ensp;[Authenticate to destination](#authenticate-to-destination)
+
+&ensp;&ensp;[Fill in destination details](#fill-in-destination-details)
+
+&ensp;&ensp;[Activate audiences to this destination](#activate-audiences-to-the-destination)
+
+&ensp;&ensp;&ensp;&ensp;[Step 1: Select your new Destination and click Next.](#step-1-select-your-new-destination-and-click-next)
+
+&ensp;&ensp;&ensp;&ensp;[Step 2: Select any audiences you want to activate, then click Next.](#step-2-select-any-audiences-you-want-to-activate-then-click-next) 
+
+&ensp;&ensp;&ensp;&ensp;[Step 3: The next step is mapping source identifiers to the Magnite device_id identifier.](#step-3-the-next-step-is-mapping-source-identifiers-to-the-magnite-device_id-identifier)
+
+&ensp;&ensp;&ensp;&ensp;[Step 4: Confirm the Destination activation Configuration.](#step-4-confirm-the-destination-activation-configuration)
+
+[Exported data / Validate data export](#exported-data--validate-data-export)
+
+## Overview
+
+The Magnite-Adobe Experience Platform integration offers Magnite Streaming clients Real-Time and Daily Destinations to map and export audiences for targeting and activation on the Magnite Streaming platform. The following document provides sample use cases to help you better understand how and when the Real time destination should be used, as well as step-by-step instructions for destination configuration for the Real-time endpoint.
```

Comment:
Just for clarification, the name of the compnay is "Magnite Streaming", so the phrase "Magnite Streaming destination" is ambiguous, since it could be referring to 1 of the 2 Magnite Streaming destinations (real-time or daily), or be referring to a "streaming" integration on the Adobe-cdp side. We would prefer to keep the name "Magnite Streaming Real-time" as this allows us to be clear that we are referring to the real-time destination and not the daily one.

Reply:
Understood. That makes sense, just please make sure to use the same name that you will use in the destination config, including the correct casing.